src/main/java/com/oltpbenchmark/api/collectors/YuagbyteCollector.java

```diff
@@ -0,0 +1,25 @@
+package com.oltpbenchmark.api.collectors;
+
```

Comment:
Any reason to check-in this again? @shaharuk-yb @rakshitjain13

Reply:
There is no need for now as we are using a Postgres collector. I will uncheck this file.